packages/frontend/src/providers/AppProvider.tsx

```diff
@@ -182,13 +183,27 @@ export const AppProvider: FC = ({ children }) => {
         },
         [showToastSuccess],
     );
+
+    const showToastInfo = useCallback<AppContext['showToastInfo']>(
+        (props) => {
+            showToastSuccess({
+                intent: Intent.NONE,
+                icon: 'info-sign',
+                timeout: 0,
```

Comment:
This should be a separate argument, rather than having the timeout being directly related to showing an info toast